lib/versioning/cargo/index.ts

```diff
@@ -9,7 +9,7 @@ export const urls = [
   'https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html',
 ];
 export const supportsRanges = true;
-export const supportedRangeStrategies = ['bump', 'extend', 'pin', 'replace'];
+export const supportedRangeStrategies = ['bump', 'widen', 'pin', 'replace'];
```

Comment:
```suggestion
export const supportedRangeStrategies: RangeStrategy = ['bump', 'widen', 'pin', 'replace'];
```
import from `lib/types`

https://github.com/renovatebot/renovate/blob/564d0fec9eedd5faefae5158535d0167dab656aa/lib/types/versioning.ts#L1

Please add that to all those `supportedRangeStrategies` definitions. That way we can enforce correct values.

Reply:
TS2322: Type 'string[]' is not assignable to type 'RangeStrategy'.

Reply:
Add a `as RangeStrategy`